src/version/cef_version_win.h

```diff
@@ -35,16 +35,28 @@
 #ifndef CEF_INCLUDE_CEF_VERSION_H_
 #define CEF_INCLUDE_CEF_VERSION_H_
 
-#define CEF_VERSION "3.3029.1604.g364cd86"
+//#define CEF_VERSION "3.3029.1604.g364cd86"
+//#define CEF_VERSION_MAJOR 3
+//#define CEF_COMMIT_NUMBER 1604
+//#define CEF_COMMIT_HASH "364cd863b14617f21e6bf8fd3ca3faea01e2ef38"
+//#define COPYRIGHT_YEAR 2017
+//
+//#define CHROME_VERSION_MAJOR 58
+//#define CHROME_VERSION_MINOR 0
+//#define CHROME_VERSION_BUILD 3029
+//#define CHROME_VERSION_PATCH 81
+
+#define CEF_VERSION "3.2987.1601.gf035232"
 #define CEF_VERSION_MAJOR 3
-#define CEF_COMMIT_NUMBER 1604
```

Comment:
You can't modify this file in this PR, this was explained by me in the other comment